src/tennisSimulator/PlayersDatabase.java

```diff
@@ -6,12 +6,24 @@
 
 public class PlayersDatabase {
 
-    private List<Player> players;
+    private static PlayersDatabase instance;
```

Comment:
final, we  don't want value to change after initialization

Reply:
initialized `PlayerDatabase`field straight after declaration.